src/altinn-app-frontend/src/utils/appUrlHelper.ts

```diff
@@ -11,67 +11,59 @@ export const profileApiUrl = `${appPath}/api/v1/profile/user`;
 export const oldTextResourcesUrl = `${origin}/${org}/${app}/api/textresources`;
 export const applicationMetadataApiUrl = `${appPath}/api/v1/applicationmetadata`;
 export const applicationSettingsApiUrl = `${appPath}/api/v1/applicationsettings`;
-export const updateCookieUrl = (partyId: string) =>
-  `${appPath}/api/v1/parties/${partyId}`;
 export const invalidateCookieUrl = `${appPath}/api/authentication/invalidatecookie`;
 export const validPartiesUrl = `${appPath}/api/v1/parties?allowedtoinstantiatefilter=true`;
 export const currentPartyUrl = `${appPath}/api/authorization/parties/current?returnPartyObject=true`;
 export const instancesControllerUrl = `${appPath}/instances`;
 export const refreshJwtTokenUrl = `${appPath}/api/authentication/keepAlive`;
 
-export function textResourcesUrl(language: string) {
-  return `${origin}/${org}/${app}/api/v1/texts/${language}`;
-}
+export const updateCookieUrl = (partyId: string) =>
+  `${appPath}/api/v1/parties/${partyId}`;
 
-export function fileUploadUrl(attachmentType: string) {
-  return (
-    `${appPath}/instances/` +
-    `${altinnWindow.instanceId}/data?dataType=${attachmentType}`
-  );
-}
+export const textResourcesUrl = (language: string) =>
+  `${origin}/${org}/${app}/api/v1/texts/${language}`;
 
-export function fileTagUrl(dataGuid: string) {
-  return (
-    `${appPath}/instances/` + `${altinnWindow.instanceId}/data/${dataGuid}/tags`
-  );
-}
+export const fileUploadUrl = (attachmentType: string) =>
+  `${appPath}/instances/${altinnWindow.instanceId}/data?dataType=${attachmentType}`;
 
-export function dataElementUrl(dataGuid: string) {
-  return `${appPath}/instances/${altinnWindow.instanceId}/data/${dataGuid}`;
-}
+export const fileTagUrl = (dataGuid: string) =>
+  `${altinnWindow.instanceId}/data/${dataGuid}/tags`;
```

Comment:
Most of the changes in this file feel pointless, and cause a lot of noise so it's almost impossible to see any _actual_ changes. In addition it introduces lots of opportunities for errors, like this case here where part of the uri is missing.